src/pages/dashboard/hooks/useDashboardData.ts

```diff
@@ -87,11 +74,29 @@ export const useDashboardData = (filters: any) => {
 
   const { showLoader, hideLoader } = useLoading();
 
-  const { data: jobsCreatedData, isLoading: isLoadingJobsCreated } = useGetV2DashboardJobsCreated<any>(
-    { queryParams: queryParams },
-    { enabled: !!filters && !filters.uuid }
+  const dashboardV3Filter = useMemo(
```

Comment:
It would be useful to type this so you get some assistance from the compiler:
```
const dashboardV3Filter = useMemo<HookFilters<typeof useJobsCreated>>(
```